.github/workflows/build.yml

```diff
@@ -156,6 +160,10 @@ jobs:
                 name: Run PHPStan
                 run: vendor/bin/phpstan analyse -c phpstan.neon -l max src/
 
+            -
+                name: Validate database schema
```

Comment:
Maybe with this occasion move right it after "Prepare test application database", where it should've been in the first place.

Reply:
Still, the PR will be green after the next v1.11 release